apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/connectionurl/parser/MysqlURLParser.java

```diff
@@ -99,11 +110,13 @@ public ConnectionInfo parse() {
         } else {
             String[] hostAndPort = hostSegment[0].split(":");
             if (hostAndPort.length != 1) {
-                return new ConnectionInfo(component, dbType, hostAndPort[0], Integer.valueOf(hostAndPort[1]), fetchDatabaseNameFromURL(location
-                        .endIndex()));
+                return new ConnectionInfo(component, dbType, hostAndPort[0], Integer.valueOf(hostAndPort[1]),
+                                          fetchDatabaseNameFromURL(location
+                                                                           .endIndex()));
             } else {
-                return new ConnectionInfo(component, dbType, hostAndPort[0], DEFAULT_PORT, fetchDatabaseNameFromURL(location
-                        .endIndex()));
+                return new ConnectionInfo(component, dbType, hostAndPort[0], DEFAULT_PORT,
+                                          fetchDatabaseNameFromURL(location
+                                                                           .endIndex()));
```

Comment:
Do you mis-format these lines? They seem no change. There is a style file in the root folder, please import them.

Reply:
ok